website/pages/docs/platform/k8s/helm/examples/enterprise-dr-with-raft.mdx

```diff
@@ -0,0 +1,193 @@
+---
+layout: "docs"
+page_title: "Highly Available Vault Enterprise Disaster Recovery Clusters with Raft"
+sidebar_current: "docs-platform-k8s-examples-ha-enterprise-dr-with-raft"
+sidebar_title: "Highly Available Vault Enterprise Disaster Recovery Clusters with Raft"
```

Comment:
Might want to make the sidebar_title a little shorter? I remember the other example page titles getting pretty squished in the sidebar.

Reply:
Yeah it's a little long, will shorten!

Reply:
Done.